public/422.html

```diff
@@ -1,67 +1,156 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title>The change you wanted was rejected (422)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
+  <title>422: Unprocessable Entity</title>
   <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
+  body{
+    margin: 0px;
   }
+  .center {
```

Comment:
For some reason, this doesn't fill my screen.  I get weird white bars on the right and bottom, no matter how I resize.  The bug is only on this page.  The other pages are fine.

Reply:
It's gone.  I think it was actually a bug in the dev tools responsive layout view.